### DISCUSSION TITLE
When trying to debug the FE in IDEA, an error occurred.

### DISCUSSION THREAD

**Discussion by xingtanzjr** [apache/doris#7835] [RFC]
（新人...）

按照官网的方法，配置了IDEA对FE的环境，`mvn generate-source`成功执行，

尝试Run PaloFE.java 的时候，抛了下面的异常，似乎是没有找到`action_table.dat`这个文件，但其实我发现它是在目录里的，有小伙伴遇到吗？

![image](https://user-images.githubusercontent.com/18027703/150505199-8ac58292-c2ef-4aa5-adb7-c0f94a08f75f.png)

**Comment by zyw8136** [comment]
action_table.dat文件在generated-sources中，没有在对应的class目录下，你需要把它拷到对应的文件夹中就好了，就可以debug了。

**Reply by binary2010** [reply]
cp fe-core/target/generated-sources/cup/org/apache/doris/analysis/action_table.dat fe-core/target/classes/org/apache/doris/analysis
